larray/core/array.py

```diff
@@ -3168,10 +3168,14 @@ def indicesofsorted(self, axis=None, kind='quicksort', reverse=False):
         """
         if axis is None:
             if self.ndim > 1:
-                raise ValueError("array has ndim > 1 and no axis specified for posargsort")
+                raise ValueError("array has ndim > 1 and no axis specified for indicesofsorted")
```

Comment:
Nice catch! The corresponding message in labelsofsorted needs to be fixed too, though.